lib/fluent/plugin/out_redis.rb

```diff
@@ -29,8 +30,13 @@ def configure(conf)
     def start
       super
 
-      @redis = Redis.new(:host => @host, :port => @port,
-                         :thread_safe => true, :db => @db_number)
+      options = {:host => @host, :port => @port,
+                 :thread_safe => true, :db => @db_number}
```

Comment:
And I've found that Fluentd v0.10.44 did not specify `required_ruby_version`: https://rubygems.org/gems/fluentd/versions/0.10.44
So, we should support Ruby 1.8.7 in this plugin, though.... :confounded: